Web/src/components/games/PandaSequence/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useEffect } from 'react';
+import { share } from 'rxjs/operators';
+import { Sequence, PandaSequenceMode } from './Sequence';
+import { createTimerObservable } from '../rxhelpers';
+import { GameState } from '../BalloonShake/GameStates';
+import SequenceDisplay from './SequenceDisplay';
+import UseSeqGenerator from './useSeqGenerator';
+
+
+const GAME_TIME = 25;
+const SEED = '100';
+const NUM_POTS = 5;
+
```

Comment:
```suggestion
```
hmmm actually are we leaving 1 or 2 blank lines ah hahaha

Reply:
Do we need to standardise?